orchestrator/graphql/resolvers/subscription.py

```diff
@@ -119,10 +130,7 @@ async def resolve_subscriptions(
     graphql_subscriptions = []
```

Comment:
I guess this is to avoid an if/else? Personally I dislike initialising variables and the overwriting them in the next line. Too many times bitten in my life with this mutable code.

Reply:
I don't see a reason to add the else only for a variable assignment, how do/did you get bitten by this?

if the logic in the if statement was more then 2 lines, I would refactor it into a function that returns the subscriptions or an empty list.

Reply:
You would be amazed I have been bitten while using mutable variables in the past 30 years ;) Mostly when the initialisation starts to move during refactoring to a different place, etc. etc. Feel free to leave it like this.